hassfest failures on PRs

### The problem

Error: R] [SERVICES] Invalid services.yaml: Services do not support device filters on target, use a device selector instead for dictionary value @ data['start_timer']['target']. Got {'device': [{'integration': 'pura', 'model': 'Pura 3'}, {'integration': 'pura', 'model': 'Pura 4'}, {'integration': 'pura', 'model': 'Pura Plus'}, {'integration': 'pura', 'model': 'Pura Mini'}], 'entity': [{'integration': 'pura'}]}
Warning: G] [CONFIG_SCHEMA] Integrations which implement 'async_setup' or 'setup' must define either 'CONFIG_SCHEMA', 'PLATFORM_SCHEMA' or 'PLATFORM_SCHEMA_BASE'. If the integration has no configuration parameters, can only be set up from platforms or can only be set up from config entries, one of the helpers cv.empty_config_schema, cv.platform_only_config_schema or cv.config_entry_only_config_schema can be used.

Error: Process completed with exit code 1.

### What version of Home Assistant Core are you running?

N/A

### What was the last working version of Home Assistant Core?

N/a

### Diagnostics information

_No response_

### Anything in the logs that might be useful?

```txt

```

### Additional information

PR pipelines are failing due to changes to hassfest

--- a/custom_components/pura/__init__.py
+++ b/custom_components/pura/__init__.py
@@ -13,6 +13,7 @@ from homeassistant.const import (
 )
 from homeassistant.core import Event, HomeAssistant, callback
 from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+from homeassistant.helpers import config_validation as cv
 from homeassistant.helpers.device_registry import DeviceEntry
 from homeassistant.helpers.typing import ConfigType
 
@@ -32,6 +33,8 @@ PLATFORMS = [
     Platform.SWITCH,
     Platform.UPDATE,
 ]
+
+CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
 
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:

--- a/custom_components/pura/services.yaml
+++ b/custom_components/pura/services.yaml
@@ -2,18 +2,14 @@
 
 start_timer:
   target:
-    device:
-      - integration: pura
-        model: Pura 3
-      - integration: pura
-        model: Pura 4
-      - integration: pura
-        model: Pura Plus
-      - integration: pura
-        model: Pura Mini
     entity:
       - integration: pura
   fields:
+    device_id:
+      required: false
+      selector:
+        device:
+          integration: pura
     slot:
       required: false
       example: 1

